projects/gigya/src/cms-components/gigya-raas/gigya-raas.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { BaseSiteService, WindowRef } from '@spartacus/core';
+import { CmsComponentData } from '@spartacus/storefront';
+import { GigyaConfig } from '../../config';
+import { GigyaRaasComponentData } from '../cms.model';
+import { gigyaProfileUpdate } from '../gigya-js/gigya-js.service';
+
+@Component({
+  selector: 'cx-gigya-raas',
+  templateUrl: './gigya-raas.component.html',
+  styleUrls: ['./gigya-raas.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class GigyaRaasComponent {
```

Comment:
`Raas` is it a abbrevation for something? Just curious

Reply:
Raas stands for Registration as a service which is a capability provided by CDC to enable configuring and displaying screen sets for registration, login, etc. on various HTML pages. We utilize this capability in our integration as well.